swapserverrpc/staticaddr.proto

```diff
@@ -74,10 +74,16 @@ message ServerWithdrawRequest {
 
     // The address that the client wants to sweep the static address deposits
     // to.
-    string client_sweep_addr = 3;
+    string client_withdrawal_addr = 3;
 
-    // The fee rate in sat/kw that the client wants to use for the sweep.
-    uint64 tx_fee_rate = 4;
+    // The amount that the client wants to withdraw, in satoshis. This amount
+    // can be a fraction of the total deposit amount.
+    int64 withdraw_amount = 4;
+
+    // The change amount will be sent back to the static address. The client
+    // ensures that the change amount covers the transaction fee, and that the
+    // change after fees is at least the dust limit.
+    int64 change_amount = 5;
```

Comment:
I propose to replace `client_withdrawal_addr`, `withdraw_amount` and `change_amount` with a psbt, so the UTXO could be used for whatever purpose, e.g. batch sending, coinjoin/payjoin, channel opening etc.